### PR TITLE
refactor: refine `cls` helper function

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

- Currently `cls` function and `classnames` function are used together, which is redundant.
- So I changed `cls`'s interface like `classnames` and usages their comsumers.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by dongsik-yoo — COMMENTED** [review]
리뷰 완료합니다. 수고하셨습니다.

File: apps/calendar/test/app/util/cssHelper.spec.ts [hunk 1]
```diff
@@ -1,23 +1,40 @@
 import { cls, CSS_PREFIX, matchViewIDRegExp } from '@src/util/cssHelper';
 
 describe('cssHelper', () => {
-  it('cls() returns css selector with prefix "tui-calendar"', () => {
-    expect(cls('view')).toBe(`${CSS_PREFIX}view`);
-    expect(cls('.layout')).toBe(`.${CSS_PREFIX}layout`);
-  });
+  describe('cls', function () {
+    it('should return css selector with prefix "tui-calendar"', function () {
```
Comment (lja1018) [inline apps/calendar/test/app/util/cssHelper.spec.ts:5]:
```suggestion
    it('should return css selector with prefix "tui-calendar"', () => {
```
화살표함수는 어떠신가요?

File: apps/calendar/test/app/util/cssHelper.spec.ts [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+        `${CSS_PREFIX}${strClassName} ${CSS_PREFIX}a ${CSS_PREFIX}c`
+      );
+    });
   });
 
-  it('matchViewIDRegExp() matches css selector with view id', () => {
-    expect(matchViewIDRegExp('allday', `${CSS_PREFIX}weekday tui-view-67`)).not.toBeNull();
-    expect(matchViewIDRegExp('daygrid', `${CSS_PREFIX}weekday tui-view-327`)).not.toBeNull();
-    expect(matchViewIDRegExp('time', `${CSS_PREFIX}time-date tui-view-1`)).not.toBeNull();
+  describe('matchViewIDRegExp', function () {
```
Comment (lja1018) [inline apps/calendar/test/app/util/cssHelper.spec.ts:33]: 바꾼다면 여기도요!

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/calendar/test/app/util/cssHelper.spec.ts (hunk 1 above)
Comment (adhrinae) [reply apps/calendar/test/app/util/cssHelper.spec.ts:5]: 어라 IntelliJ 기본 스니펫이 이렇게 되어있어서 그랬나봐요.
저장할때 ESLint Fix 됐을줄알았는데 안됐네

**Pull request merged**